youtube_dl/extractor/wikimedia.py

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..utils import get_element_by_class, determine_ext, clean_html, KNOWN_EXTENSIONS
```

Comment:
Preferably per line (makes it easier to see changes) and sorted:
```suggestion
from ..utils import (
    clean_html,
    determine_ext,
    get_element_by_class,
    KNOWN_EXTENSIONS
)
```
You'll also have to add some imports from `..compat` if other suggestions are accepted.